### PR TITLE
Matrix Addition in Java Added #727

### PR DESCRIPTION
**Pull request by krishna-NIT**
#727 Addition of Matrix Done in JAVA
@plazzy99  @OjusWiZard @nakul-19 @rudrakshi99  
Please Review It

### CODE REVIEW

**Comment by krishna-NIT** [comment]
@rish-singhal I have done my Corrections, Now it is in Alphabetical Order, Please Review it and Please Tell me changes if Required

**Comment by krishna-NIT** [comment]
@Rohan27s @tarun26091999  Please Review it and Let me Know if any changes

**Comment by krishna-NIT** [comment]
@rish-singhal  No Conflicts Right now Sir, Done

File: 2D_Array/readme.md [hunk 1]
```diff
@@ -9,15 +9,16 @@ Due to the fact that the elements of 2D arrays can be random accessed. Similar t
...
 * Spiral Print ----> [C++](/Code/C++/spiral_print.cpp)
 * Staircase Search ----> [C++](/Code/C++/staircase_search.cpp)
+* Transpose of a Matrix --->[Java](/Code/Java/transpose.java)|[Python](/Code/Python/Transpose_of_matrix.py)
+* Wave Print ----> [C++](/Code/C++/wave_print.cpp)
 * Transpose of a Matrix ----> [Python](/Code/Python/Transpose_of_matrix.py)
 * Wave Print ----> [C++](/Code/C++/wave_print.cpp)
-* Row wise sum ---->[java](/Code/java/RowWise_Sum.java)
-
-
+* Row wise sum ---->[java](/Code/java/RowWise_Sum.java)
```
Comment (rish-singhal) [inline 2D_Array/readme.md:24]: Why this row wise sun there at last?

**Review by rish-singhal — CHANGES_REQUESTED** [review]

File: 2D_Array/readme.md (hunk 1 above)
Comment (krishna-NIT) [reply 2D_Array/readme.md]: Sorry SIr, Missed it, Done Sir

**Comment by krishna-NIT** [comment]
@rish-singhal Done Sir

**Pull request merged**